server/helpers/signin.js

```diff
@@ -6,7 +6,7 @@ const login = {
     const loginvalidation = {
```

Comment:
I think the best thing to do in this case would be to check if the password/email matches not validating it again! Why are you validating them again?

Reply:
Oh thank you so mush for such feedback, that is true! i should not be validating a user for signing him/her in. Thanks!